### PR TITLE
[DOCFIX] Update comment for ALLUXIO_MASTER_VERSION_SHORT

### PR DESCRIPTION
**Pull request by waltonryan**
To better reflect convention of `ALLUXIO_MASTER_VERSION_SHORT`

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ kramdown:
 
 # These allow the documentation to be updated with new releases of Alluxio.
 ALLUXIO_RELEASED_VERSION: 1.2.0
-# This would be "master" for the master branch, 1.1.1 for the 1.1.1 release, and
-# 1.1.1-RC2 for the second 1.1.1 release candidate.
+# This would be "master" for the master branch, 1.1 for the 1.1.1 release, and
+# 1.1 for the second 1.1.1 release candidate.
```
Comment (calvinjia) [inline docs/_config.yml]: Could this be reworded to:

`For example, "master" for the master branch, and 1.1 for 1.1.x releases and release candidates`

**Comment by waltonryan** [comment]
Addressed comment and updated.

File: docs/_config.yml (hunk 1 above)
Comment (waltonryan) [inline docs/_config.yml]: sgtm. Updated.

**Comment by calvinjia** [comment]
LGTM thanks

**Pull request merged**